lib/fastlane/plugin/yarn/actions/yarn_action.rb

```diff
@@ -2,18 +2,25 @@ module Fastlane
   module Actions
     class YarnAction < Action
       def self.run(params)
-        task = params[:task]
+        flags = params[:flags]
         command = params[:command]
+        options = params[:options]
         package_path = params[:package_path]
 
         # create a new object from yarn helper
         yarn = Helper::YarnHelper.new(package_path: package_path)
 
         # Check if yarn is installed
         yarn.check_install
+        yarn.install_dependencies
```

Comment:
I'm still on the fence a bit around whether the command should be installing dependencies on every command run.

I can see the value in doing this, but it seems like we'll be wasting a few seconds on every command by doing this.